racket/src/rktio/rktio_fs.c

```diff
@@ -633,6 +633,7 @@ char *rktio_get_current_directory(rktio_t *rktio)
    free(s);
    get_posix_error();
  }
+ free(s);
```

Comment:
Did you mean for this `free` to be here?

Reply:
Aha, that explains why this doesn't work. Thanks!

That free was about another coverity report, but that one is a false positive because getcwd is terrible.